src/app/App.js

```diff
@@ -13,16 +14,19 @@ class App {
     this.modal = new Modal('modalBox');
     this.mainMenuPanel = new MainMenu('mainMenu', ['People', 'Vehicles', 'Starships'], 0);
     this.whiteButton = new WhiteButton('whiteButton', 'Hall of fame', 'whiteButton');
-    
     this.whiteButton.addIcon('../../static/assets/ui/hof.svg');
-
+    this.rules = new Rules('Mode Rules', 'rules');
     this.btns = document.querySelectorAll('.mainMenu > div > button');
-    this.btns.forEach(btn => {
-      btn.addEventListener('click', (e) => {
-        this.mainMenuPanel.btnIndex(e, this.btns)
+    this.mainMenuPanel.addClasses(this.mainMenuPanel.gameModeIndex, this.btns);
```

Comment:
Dlaczego to przenieśliśmy tutaj?

Reply:
Pisałem o tym przedwczoraj na discordzie, - pogadamy na daily,